src/components/inventoryList/inventoryList.js

```diff
@@ -113,21 +113,29 @@ class InventoryList extends React.Component {
         session
       });
 
-      const hasGuests = cellData?.numberOfGuests > 0;
-      const guestsId = cellData?.subscriptionManagerId;
       updatedColumnHeaders = columnHeaders;
 
+      const guestsId = cellData?.subscriptionManagerId;
+      let hasGuests = cellData?.numberOfGuests > 0 && guestsId;
+
+      // Apply hasGuests callback, return boolean
+      if (typeof settings.hasGuests === 'function') {
```

Comment:
required:
should convert this to `settings?.hasGuests` since it's not a required prop